tasks/systemd-zerg.yml

```diff
@@ -0,0 +1,78 @@
+---
```

Comment:
Maybe drop this and the zergling/zergpool unit files entirely? As we'll be moving away from uWSGI anyway, so are unlikely to ever put in the effort to make this work.

Reply:
yeah, that's a very fair point.